contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1090,6 +1022,97 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         );
     }
 
+    /**
+     * Handles the creation-specific safety measures required for OVM contract deployment.
+     * This function sanitizes the return types for creation messages to match calls (bool, bytes).
+     * This allows for consistent handling of both types of messages in _handleExternalMessage().
+     * 
+     * @param _gasLimit Amount of gas to be passed into this creation.
+     * @param _creationCode Code to pass into CREATE for deployment.
+     * @param _address OVM address being deployed to.
+     * @return _success Whether or not the call succeeded.
+     * @return _returndata If creation fails: revert data.  Otherwise: empty.
+     */
+    function _handleExternalCreate(
+        uint _gasLimit,
+        bytes memory _creationCode,
+        address _address
+    )
+        internal
+        returns(
+            bool,
+            bytes memory
+        )
+    {
+        // Check that there is not already code at this address.
+        if (_hasEmptyAccount(_address) == false) {
+            return (
+                false,
+                _encodeRevertData(
```

Comment:
This differs from the EVM's behavior. Create collision uses all provided gas and returns no data: https://github.com/ethereumjs/ethereumjs-monorepo/blob/53150c159b559b31b6b7c321d784dbb11b082fb9/packages/vm/lib/evm/evm.ts#L270-L278

Reply:
Gonna leave it; OOG may cause worse devex due to nuisance gas usage.

Reply:
Would you mind leaving a comment that explains this for future readers? ty ser